### PR TITLE
ENCD-4407-4497-master

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/encoded/schemas/matched_set.json [hunk 1]
```diff
@@ -87,19 +87,22 @@
             "title": "Biosample treatment"
         },
         "related_datasets.files.file_type": {
-            "title": "Available data"
+            "title": "Available data types"
```
Comment (gabdank) [inline src/encoded/schemas/matched_set.json]: Should be "Available file types"

File: src/encoded/schemas/file.json [hunk 2]
```diff
@@ -862,7 +862,7 @@
             "pattern": "[a-f\\d]{32}|[A-F\\d]{32}"
         },
         "content_md5sum": {
-            "title": "Content MD5sum",
+            "title": "Content mD5sum",
```
Comment (gabdank) [inline src/encoded/schemas/file.json]: Should be "Content MD5sum"

File: src/encoded/schemas/page.json [hunk 3]
```diff
@@ -197,16 +197,13 @@
             "title": "Topics"
         },
         "month_released": {
-            "title": "Date"
+            "title": "Date released"
```
Comment (gabdank) [inline src/encoded/schemas/page.json]: I think this should be "Month released"

File: src/encoded/schemas/treatment.json [hunk 4]
```diff
@@ -34,20 +34,20 @@
...
             "rdfs:subPropertyOf": "rdfs:seeAlso",
-            "title": "External identifiers",
+            "title": "External resources",
             "description": "Identifiers from external resources that may have 1-to-1 or 1-to-many relationships with ENCODE objects.",
             "type": "array",
             "uniqueItems": true,
             "default": [],
             "items": {
-                "title": "External identifier",
+                "title": "External identifiers",
```
Comment (gabdank) [inline src/encoded/schemas/treatment.json]: Should be "External identifier"

File: src/encoded/schemas/target.json [hunk 5]
```diff
@@ -89,13 +89,13 @@
...
         "dbxref": {
-            "title": "External identifiers",
+            "title": "External resources",
             "description": "Unique identifiers from external resources (e.g. HGNC, GeneID, UniProtKB or ENSEMBL).",
             "type": "array",
             "default": [],
             "uniqueItems": true,
             "items": {
-                "title": "External identifier",
+                "title": "External identifiers",
```
Comment (gabdank) [inline src/encoded/schemas/target.json]: Should be "External identifier"

File: src/encoded/schemas/worm_donor.json [hunk 6]
```diff
@@ -85,25 +85,22 @@
...
+            "title": "External resources",
             "description": "Identifiers from external resources that may have 1-to-1 or 1-to-many relationships with ENCODE objects.",
             "comment": "Do not submit. DCC personnel is responsible for submission of donor external resource identifiers.",
             "permission": "import_items",
             "type": "array",
             "default": [],
             "uniqueItems": true,
             "items": {
-                "title": "Database external identifier",
+                "title": "External resources",
```
Comment (gabdank) [inline src/encoded/schemas/worm_donor.json]: Should be "External identifier"

Comment (gabdank) [inline src/encoded/schemas/worm_donor.json]: Should be "External identifier"

File: src/encoded/schemas/aggregate_series.json [hunk 7]
```diff
@@ -163,43 +162,43 @@
...
+            "title": "Organism",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.life_stage": {
             "title": "Life stage",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.age_display": {
-            "title": "Age display",
+            "title": "Biosample ages",
```
Comment (gabdank) [inline src/encoded/schemas/aggregate_series.json]: Should be "Biosample age"

File: src/encoded/schemas/matched_set.json [hunk 8]
```diff
@@ -165,43 +164,43 @@
...
+            "title": "Organism",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.life_stage": {
             "title": "Life stage",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.age_display": {
-            "title": "Age display",
+            "title": "Biosample ages",
```
Comment (gabdank) [inline src/encoded/schemas/matched_set.json]: Should be "Biosample age"

File: src/encoded/schemas/organism_development_series.json [hunk 9]
```diff
@@ -153,43 +152,43 @@
...
             "type": "string"
         },
         "related_datasets.replicates.library.biosample.organism.scientific_name": {
-            "title": "Species",
+            "title": "Organism",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.age_display": {
-            "title": "Age display",
+            "title": "Biosample ages",
```
Comment (gabdank) [inline src/encoded/schemas/organism_development_series.json]: Should be "Biosample age"

File: src/encoded/schemas/pipeline.json [hunk 10]
```diff
@@ -196,30 +196,30 @@
     },
     "facets": {
         "assay_term_names": {
-            "title": "Assay"
+            "title": "Assay names"
```
Comment (gabdank) [inline src/encoded/schemas/pipeline.json]: Should be "Assay name"

File: src/encoded/schemas/pipeline.json [hunk 11]
```diff
@@ -196,30 +196,30 @@
...
         "accession": {
             "title": "Accession"
         },
         "title": {
-            "title": "Pipeline"
+            "title": "Title"
         },
         "assay_term_names": {
-            "title": "Assay type"
+            "title": "Assay names"
```
Comment (gabdank) [inline src/encoded/schemas/pipeline.json]: Should be "Assay name"

File: src/encoded/schemas/pipeline.json [hunk 12]
```diff
@@ -228,19 +228,16 @@
             "title": "Status"
         },
         "analysis_steps.versions.software_versions.downloaded_url": {
-            "title": "URL"
+            "title": "Software URLs"
```
Comment (gabdank) [inline src/encoded/schemas/pipeline.json]: Should be "Software URL"

File: src/encoded/schemas/reference_epigenome.json [hunk 13]
```diff
@@ -175,47 +174,47 @@
...
+            "title": "Organism",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.life_stage": {
             "title": "Life stage",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.age_display": {
-            "title": "Age display",
+            "title": "Biosample ages",
```
Comment (gabdank) [inline src/encoded/schemas/reference_epigenome.json]: Should be "Biosample age"

File: src/encoded/tests/features/search.feature [hunk 14]
```diff
@@ -79,8 +79,8 @@ Feature: Search
         When I press "Data"
         And I click the link to "/search/?type=Experiment&status=released"
         And I wait for the content to load
-        When I fill in "searchAssay" with "dna"
-        Then I should see at least 2 elements with the css selector "div.term-list.searchAssay > li"
+        When I fill in "searchAssaytitle" with "dna"
```
Comment (caseylitton) [inline src/encoded/tests/features/search.feature]: flagging so we don't forget to squash this commit.

File: src/encoded/schemas/replication_timing_series.json [hunk 15]
```diff
@@ -187,43 +186,39 @@
...
-            "title": "Species",
-            "type": "array"
-        },
         "related_datasets.replicates.library.biosample.life_stage": {
             "title": "Life stage",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.age_display": {
-            "title": "Age display",
+            "title": "Biosample ages",
```
Comment (gabdank) [inline src/encoded/schemas/replication_timing_series.json]: Should be "Biosample age"

File: src/encoded/schemas/target.json (hunk 5 above)
Comment (gabdank) [inline src/encoded/schemas/target.json]: Should be "External identifier"

File: src/encoded/schemas/treatment_concentration_series.json [hunk 16]
```diff
@@ -159,43 +158,43 @@
...
+            "title": "Organism",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.life_stage": {
             "title": "Life stage",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.age_display": {
-            "title": "Age display",
+            "title": "Biosample ages",
```
Comment (gabdank) [inline src/encoded/schemas/treatment_concentration_series.json]: Should be "Biosample age"

File: src/encoded/schemas/treatment_time_series.json [hunk 17]
```diff
@@ -163,43 +162,43 @@
...
+            "title": "Organism",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.life_stage": {
             "title": "Life stage",
             "type": "array"
         },
         "related_datasets.replicates.library.biosample.age_display": {
-            "title": "Age display",
+            "title": "Biosample ages",
```
Comment (gabdank) [inline src/encoded/schemas/treatment_time_series.json]: Should be "Biosample age"

**Pull request merged**